Makefile

```diff
@@ -24,6 +24,7 @@ MCUBOOT_VERSION ?= $(shell cat $(ROOT_DIR)/config/mcuboot_version)
 BAUDRATE        ?= 115200
 BUILD_TYPE      ?= Release
 TARGET_BOARD    ?= LEKA_V1_2_DEV
+FIRMWARE_VERSION?= 1.1.0
```

Comment:
using the make file to run the script is a good idea, but the makefile itself cannot define the version.

just like for mbed-os version, the firmware version must live in a separate file like  `config/leka_os_version`

then use the `FIRMWARE_VERSION?=$(shell cat $(ROOT_DIR)/config/leka_os_version)`

Reply:
ok